src/data/cognito.ts

```diff
@@ -46,3 +48,21 @@ export const listUsersInPool = async (
 
     return users;
 };
+
+const generateTemporaryPassword = (): string =>
+    Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
+
+export const addUserToPool = async (
+    cognito: CognitoIdentityServiceProvider,
+    userPoolId: string,
+    formUser: FormUser,
+): Promise<void> => {
+    const params: AdminCreateUserRequest = {
+        UserPoolId: userPoolId,
+        Username: formUser.email,
+        UserAttributes: [{ Name: 'custom:noc', Value: formUser.nocs }],
```

Comment:
Just realised something else the script does is set the `email_verified` attribute to `true` which is needed to send password reset emails so this will need to be added in to the list of attributes here